priiloader/include/hacks.h

```diff
@@ -34,6 +34,8 @@ struct system_patch {
 
 struct system_hack {
 	std::string desc;
+	std::string provide;
+	std::string require;

```

Comment:
i suggest a rename here. provide and require are confusing.
i suggest a name like MasterID & RequiredID or something.

with only a name and context , you should be able to say what is what, and i can not do that here.

Reply:
I thought of these as "a hack provides a certain feature" or "a hack requires a certain other feature". But I can rename these to MasterID and RequiredID or something if you want.

Reply:
Renamed them to `masterID` and requiredMasterID`.